File: Linguagem C++/Estrutura Condicional/Operadora.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{

    int qnt;
    double valorTotal;

    cout << "Digite a quantidade de minutos: ";
    cin >> qnt;

    cout << fixed << setprecision(2);
    if (qnt <= 100){
        valorTotal = 50.00;
    }else {
        valorTotal = ((qnt - 100) * 2.00) + 50.00;
    };
    cout << "Valor a pagar: " << valorTotal << endl;

    return 0;
}

File: Linguagem Java/Exercicios/Operadora.java
import java.util.Locale;
import java.util.Scanner;

public class Operadora {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int qnt;
		double valorTotal;
		
		System.out.print("Digite a quantidade de minutos: ");
		qnt = sc.nextInt();
		
		if (qnt <= 100) {
			valorTotal = 50.00;			
		}else{
			valorTotal = ((qnt - 100) * 2.00) + 50.00;			
		};
		
		System.out.println("Valor a pagar: R$ "+String.format("%.2f", valorTotal));
		
		sc.close();
		
	}

}
